drivers/hmis/app/graphql/types/hmis_schema/user.rb

```diff
@@ -11,12 +11,20 @@ class HmisSchema::User < Types::BaseObject
     description 'HUD User'
```

Comment:
Currently we only use this type to resolve `Hmis::Hud::User`s, not `Hmis::User`s. The `Hmis::Hud::User` is something that gets attached to records to record who last changed them. The `Hmis::User` is the actual application user (`current_user`).

I think we want a separate type for the actual application user –– that's the user which will have the recent items, any information about roles and permissions, etc. We're going to need this anyway when we get to the permissions stuff, so it will be good to get it done! 

My suggestion would be to rename this type to HudUser, and add a new `Types::Application::User` type (in a new module `types/application/user.rb`) that has recent_items on it. Resolve `current_user` query to this type.

Later on we will add any permissions/roles that we want to expose to that type. @eanders does that sound reasonable to you?